ipn/ipnlocal/local.go

```diff
@@ -2201,6 +2208,17 @@ func (b *LocalBackend) setNetInfo(ni *tailcfg.NetInfo) {
 	cc.SetNetInfo(ni)
 }
 
+func hasCapability(nm *netmap.NetworkMap, cap string) bool {
+	if nm != nil && nm.SelfNode != nil && len(nm.SelfNode.Capabilities) > 0 {
```

Comment:
no need for the len check. The range on nil or empty slice is equivalent.

Reply:
oh, didn't know that `range` can be used on `nil`. will remove.